### PR TITLE
Replace tree grid FOODS_DATA with orders data

### LINKED ISSUE #2941: Replace Tree Grid FOODS_DATA sample data
FOODS_DATA is used in a lot of our tree grid sample, but it looks a bit ugly. It is not an actual homogenous tree data and doesn't make sense in a tree grid.

### PR DESCRIPTION
**Pull request by PlamenaMiteva**
Closes #2941

### CODE REVIEW

File: src/app/tree-grid/tree-grid-summary-sample/tree-grid-summary-sample.component.html [hunk 1]
```diff
@@ -22,22 +42,23 @@
                 <igx-icon class="header-icon" style.color="{{ col.hasSummary ? '#e41c77' : '' }}" (click)="toggleSummary(col)">functions</igx-icon>
             </ng-template>
         </igx-column>
-        <igx-column #col field="Discontinued" header="Discontinued" [hasSummary]="true" [dataType]="'boolean'">
+        <igx-column #col field="Delivered" header="Delivered" [hasSummar
+        y]="true" [dataType]="'boolean'">
```
[the remark this answers is not in the document]
Comment (igdmdimitrov) [reply src/app/tree-grid/tree-grid-summary-sample/tree-grid-summary-sample.component.html:46]: Also, `<igx-column #col field="ID" header="Product ID">` should be changed to `<igx-column #col field="ID" header="Order ID">`

File: src/app/tree-grid/tree-grid-excel-style-filtering-sample-1/tree-grid-excel-style-filtering-sample-1.component.html [hunk 2]
```diff
@@ -11,23 +11,24 @@
...
+        </igx-column>
+        <igx-column field="UnitPrice" header="Unit Price" [dataType]="'currency'" [pipeArgs]="formatOptions" [sortable]="true" >
+        </igx-column>
+        <igx-column field="Price" header="Price" [dataType]="'currency'" [pipeArgs]="formatOptions" [sortable]="true">
         </igx-column>
-        <igx-column field="AddedDate" header="Added Date" [dataType]="'date'" [formatter]="formatDate" [sortable]="true">
+        <igx-column field="OrderDate" header="Order Date" [dataType]="'date'" [formatter]="formatDate" [sortable]="true">
         </igx-column>
-        <igx-column field="Discontinued" header="Discontinued" [dataType]="'boolean'" [sortable]="true">
+        <igx-column field="Delivered" header="Deliverued" [dataType]="'boolean'">
```
Comment (igdmdimitrov) [inline src/app/tree-grid/tree-grid-excel-style-filtering-sample-1/tree-grid-excel-style-filtering-sample-1.component.html:28]: **Fix this in all files**

```suggestion
        <igx-column field="Delivered" header="Delivered" [dataType]="'boolean'">
```

File: src/app/tree-grid/tree-grid-conditional-cell-style-sample/tree-grid-conditional-cell-style-sample.component.html [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+        </igx-column>
+        <igx-column field="UnitPrice" header="Unit Price" [dataType]="'currency'" [pipeArgs]="formatOptions" [cellClasses] = "priceClasses">
+        </igx-column>
+        <igx-column field="Price" header="Price" [dataType]="'currency'" [pipeArgs]="formatOptions">
         </igx-column>
-        <igx-column field="AddedDate" header="Added Date" [dataType]="'date'" [formatter]="formatDate">
+        <igx-column field="OrderDate" header="Order Date" [dataType]="'date'" [formatter]="formatDate">
         </igx-column>
-        <igx-column field="Discontinued" header="Discontinued" [dataType]="'boolean'">
+        <igx-column field="Discontinued" header="Discontinued" [sortable]="true" [disableHiding]="true" [dataType]="'boolean'">
```
Comment (igdmdimitrov) [inline src/app/tree-grid/tree-grid-conditional-cell-style-sample/tree-grid-conditional-cell-style-sample.component.html:18]:
```suggestion
        <igx-column field="Delivered" header="Delivered" [sortable]="true" [disableHiding]="true" [dataType]="'boolean'">
```

File: src/app/tree-grid/tree-grid-conditional-cell-style-sample/tree-grid-conditional-cell-style-sample.component.html [hunk 4]
```diff
@@ -1,23 +1,24 @@
...
             <ng-template igxCell let-cell="cell" let-val>
-                <span *ngIf="cell.row.data.UnitPrice === 0">-</span>
-                <img *ngIf="cell.row.data.UnitPrice !== 0 && val" src="assets/images/grid/active.png" title="Continued" alt="Continued" />
-                <img *ngIf="cell.row.data.UnitPrice !== 0 && !val" src="assets/images/grid/expired.png" title="Discontinued" alt="Discontinued" />
+                <img *ngIf="val" src="assets/images/grid/active.png" title="Continued" alt="Continued" />
+                <img *ngIf="!val" src="assets/images/grid/expired.png" title="Discontinued" alt="Discontinued" />
```
Comment (igdmdimitrov) [inline src/app/tree-grid/tree-grid-conditional-cell-style-sample/tree-grid-conditional-cell-style-sample.component.html:21]:
```suggestion
                <img *ngIf="val" src="assets/images/grid/active.png" title="Delivered" alt="Delivered" />
                <img *ngIf="!val" src="assets/images/grid/expired.png" title="Undelivered" alt="Undelivered" />
```

File: src/app/tree-grid/tree-grid-summary2-sample/tree-grid-summary2-sample.component.html [hunk 5]
```diff
@@ -12,7 +12,7 @@
 
 <igx-tree-grid [igxPreventDocumentScroll]="true"  #treegrid1 [data]="data" [autoGenerate]="false" height="500px" width="100%"
     primaryKey="ID" foreignKey="ParentID">
-    <igx-column #col field="ID" header="Product ID" [sortable]="true">
+    <igx-column #col field="ID" header="Order ID" [sortable]="true">
     </igx-column>
     <igx-column #col field="Name" header="Product Name" [hasSummary]="true">
```
Comment (igdmdimitrov) [inline src/app/tree-grid/tree-grid-summary2-sample/tree-grid-summary2-sample.component.html:17]: Fix this and the `Discontinued` and `Added Date`.

```suggestion
    <igx-column #col field="Name" header="Order Name" [hasSummary]="true">
```

**Review by igdmdimitrov — CHANGES_REQUESTED** [review]

**Comment by kdinev** [comment]
@Lipata Could we change the ORDERS_DATA to not be a function and we should get this PR merged?

**Review by kdinev — DISMISSED** [review]

**Comment by kdinev** [comment]
@igdmdimitrov Could you review again?

**Review by igdmdimitrov — APPROVED** [review]

**Review by Lipata — APPROVED** [review]

**Pull request merged**